Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN apt-get update && \
       ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb && \
     apt-get update && \
     apt-get -y --no-install-recommends install libarrow-dev libprotobuf-dev && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* "apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb"
```

Comment:
Wouldn't it make sense to simple blanket delete all `.deb`s at this point? I know this is currently the only one, but `*.deb` would also make slightly easier to adjust to future upstream file name changes.

Reply:
Done. Don't really care either way.